src/planner/macos/mod.rs

```diff
@@ -72,6 +72,20 @@ pub struct Macos {
     /// The root disk of the target
     #[cfg_attr(feature = "cli", clap(long, env = "NIX_INSTALLER_ROOT_DISK"))]
     pub root_disk: Option<String>,
+
+    /// On AWS, put the Nix Store volume on the EC2 instances' instance store volume.
+    ///
+    /// Using the instance store volume bypasses the interactive "enable full disk access" step.
+    /// However, using the instance store volume means the machine can never be "Stopped".
+    /// If the instance is Stopped, the instance store volume is erased, and the installation is broken.
+    /// Note: the machine can still be rebooted.
```

Comment:
Let's move this fairly important caveat to be first in this paragraph, and have the actual "nice effect" noted after the user ostensibly had the time to read the caveat, so they can weigh their options.

Also probably expand more on the "interactive" prompt step thing -- like "without this, installations on ec2 macs need manual intervention to enable fill disk access in order to be able to (re)mount the Nix store"